src/demo/renderer.js

```diff
@@ -352,7 +351,15 @@ const drawMovingFish = state => {
       }
     }
 
-    drawSingleFish(fish, x, y, ctx, 1, drawPrediction);
+    drawSingleFish(
+      fish,
+      x,
+      y,
+      ctx,
+      1,
+      drawPrediction,
+      state.currentMode === Modes.Training
```

Comment:
Can you comment on what this line is signaling? Just something like `// drawPolaroid`